### PR TITLE
Setup `lfs-warning` GitHub Action check

### LINKED ISSUE #1370: Setup `lfs-warning` GitHub Action check
### What should be done?

We should use https://github.com/ActionsDesk/lfs-warning to ensure that images are added using git-lfs.

### Where?

https://github.com/FerretDB/FerretDB/blob/main/.github/workflows/docs.yml

### PR DESCRIPTION
**Pull request by ndkhangvl**
Closes #1370.

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -68,6 +68,14 @@ jobs:
           git status
           git diff --exit-code
 
+      - name: lfs warning
```
Comment (AlekSi) [inline .github/workflows/docs.yml:71]: Let's use a better name. All other names are properly capitalized, for example

File: .github/workflows/docs.yml [hunk 2]
```diff
@@ -68,6 +68,14 @@ jobs:
...
+        uses: actionsdesk/lfs-warning@v3.2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (AlekSi) [inline .github/workflows/docs.yml:74]: Let's remove that default value

File: .github/workflows/docs.yml [hunk 3]
```diff
@@ -68,6 +68,14 @@ jobs:
...
+          filesizelimit: 10MB
```
Comment (AlekSi) [inline .github/workflows/docs.yml:75]: Let's set to 1MB

File: .github/workflows/docs.yml [hunk 4]
```diff
@@ -68,6 +68,14 @@ jobs:
...
+          exclusionPatterns: |
+          **/*.png
```
Comment (AlekSi) [inline .github/workflows/docs.yml:77]: That's not a valid YAML syntax – the value should be indented

Comment (AlekSi) [reply .github/workflows/docs.yml:77]: Actually, let's remove them – we don't want to exclude PNGs

**Review by AlekSi — APPROVED** [review]
LGTM. Please sign our CLA so we can merge this PR: https://github.com/FerretDB/FerretDB/pull/1371#issuecomment-1303763815

**Comment by ndkhangvl** [comment]
> LGTM. Please sign our CLA so we can merge this PR: [#1371 (comment)](https://github.com/FerretDB/FerretDB/pull/1371#issuecomment-1303763815)

sorry I'm newbie, how do I sign CLA

**Comment by AlekSi** [comment]
No worries :) In [that](https://github.com/FerretDB/FerretDB/pull/1371#issuecomment-1303763815) comment click the yellow "not signed yet" button or "Contributor License Agreement" link

**Comment by ndkhangvl** [comment]
> No worries :) In [that](https://github.com/FerretDB/FerretDB/pull/1371#issuecomment-1303763815) comment click the yellow "not signed yet" button or "Contributor License Agreement" link

I did, thanks for the tutorial

**Comment by AlekSi** [comment]
Thank you! I changed that PR to skip that step until we know what permissions are required (https://github.com/ActionsDesk/lfs-warning/issues/147), but that should be easy to enable and fix later.

**Pull request merged**